assets/queries/terraform/azure/aks_uses_azure_policies_addon/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
```

Comment:
I think we can check if `addon_profile` and `azure_policy` are undefined as well (since these attributes are optional)